pkg/queue/breaker.go

```diff
@@ -51,6 +51,13 @@ func NewBreaker(queueDepth, maxConcurrency int32) *Breaker {
 // already consumed, Maybe returns immediately without calling thunk. If
 // the thunk was executed, Maybe returns true, else false.
 func (b *Breaker) Maybe(thunk func()) bool {
+
+	if b.maxConcurrency == 0 {
+		// Unlimited concurrency
+		thunk()
+		return true
+	}
```

Comment:
Shouldn't this change been taken out?

Reply:
Whoops.  I thought this would be removed when merging #1807.  Fixed.